Lab/demo/Lab/Plugins/PMP/Selection_plugin.cpp

```diff
@@ -977,13 +977,19 @@ public Q_SLOTS:
     }
     //Add vertex and face to border
     case 9:
+    {
+      QPixmap pm(":/cgal/Lab/resources/euler_deg2.png");
+      ui_widget.docImage_Label->setPixmap(pm);
+      break;
+    }
+    case 10:
     {
       QPixmap pm(":/cgal/Lab/resources/add_facet1.png");
```

Comment:
Hard coded path.

Reply:
same as the others

Reply:
So, what is it then needed for?

Reply:
it's a relative path, it's the images in the selection item.

Reply:
> Hard coded path.

It is not an hard-coded path on the filesystem. As the path starts with `:/`, for Qt it has a special meaning: the "file" is searched for in the Qt resources shipped with the binary (declared via [the `qrc` system](https://doc.qt.io/qt-6/resources.html)).

Reply:
Sorry for the noise, and thank you for the explanation.